ranger-zookeeper/src/main/java/io/appform/ranger/zookeeper/servicefinderhub/ZkServiceDataSource.java

```diff
@@ -56,6 +60,7 @@ public Collection<Service> services() {
                 .forPath(PathBuilder.REGISTERED_SERVICES_PATH);
         return null == children ? Collections.emptySet() :
                 children.stream()
+                        .filter(child -> !excludedServices.contains(child))
```

Comment:
Revert

Reply:
reverted, filtering in `ServiceFinderHub` using `getEligibleServices` method